scripts/spectrum-vars.js

```diff
@@ -320,17 +320,21 @@ async function processSpectrumVars() {
         }
 
         // fonts.css (2 sources so a little tricky)
-        {
-            // const srcPath1 = path.join(commonsPath, 'fonts.css');
-            const srcPath2 = path.join(typographyPath, 'font.css');
-            const dstPath = path.resolve(
-                path.join(__dirname, '..', 'tools', 'styles', 'fonts.css')
-            );
-            console.log(`processing fonts from commons & typography`);
-            processes.push(
-                processMultiSourceCSS([srcPath2], dstPath, ':root ')
-            );
-        }
+        // {
+        //     // const srcPath1 = path.join(commonsPath, 'fonts.css');
+        //     const srcPath2 = path.join(
+        //         typographyPath,
+        //         'dist',
+        //         'index-vars.css'
+        //     );
+        //     const dstPath = path.resolve(
+        //         path.join(__dirname, '..', 'tools', 'styles', 'fonts.css')
+        //     );
+        //     console.log(`processing fonts from commons & typography`);
+        //     processes.push(
+        //         processMultiSourceCSS([srcPath2], dstPath, ':root ')
+        //     );
+        // }
```

Comment:
If you're OK for it, we can just remove this section, and lines 303-310 which should no longer be used here.